MSAL/src/MSALPublicClientApplication.m

```diff
@@ -879,6 +879,7 @@ - (void)acquireTokenSilentWithParameters:(MSALSilentTokenParameters *)parameters
     // Nested auth protocol
     msidParams.nestedAuthBrokerClientId = self.internalConfig.nestedAuthBrokerClientId;
     msidParams.nestedAuthBrokerRedirectUri = self.internalConfig.nestedAuthBrokerRedirectUri;
+    msidParams.skipSendRedirectUri = [NSStringFromClass([self class]) isEqualToString:@"MSAL.MSALNativeAuthPublicClientApplication"];
```

Comment:
Fair point, right now is not possible to pass nil as the redirect URI, so is impossible to know for certain that the developer wants or not to use it. I'll look into your suggestion, but I think that is not the only validation that we need to surpass.

Reply:
Changing the mentioned code won't have the intended effect, since this method will return a nil instance of `MSIDRedirectUri`, which is what we want if developers don't pass a redirect URI. There are other points in MSAL and IdentityCore that are not taking in consideration the `bypassRedirectUriValidation` parameter. I think that the latest change in this PR addresses that.

Reply:
I updated the IdentityCore PR as well.